### PR TITLE
Add Website to Storefront Account embedded

### PR DESCRIPTION
**Pull request by XaosSintez**
## Summary

This PR adds Website to Storefront Account embedded

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Comment by ashkarpetin** [comment]
Please add `expand` query parameter to StorefrontGetAccount

File: openapi/components/schemas/Storefront/Account.yaml [hunk 1]
```diff
@@ -43,3 +43,13 @@ properties:
     items:
       anyOf:
         - $ref: ../Links/SelfLink.yaml
+  _embedded:
+    type: array
+    description: >-
+      Any embedded objects available that are requested by the `expand`
+      querystring parameter.
```
Comment (noon-dawg) [inline openapi/components/schemas/Storefront/Account.yaml:50]: nitpick: I'm not sure if we need to state "any" or "available" here

These words sound a bit passive 

```suggestion
      Available embedded objects that are requested by the `expand`
      querystring parameter.
```
or
```suggestion
      Embedded objects that are requested by the `expand`
      querystring parameter.
```

**Comment by XaosSintez** [comment]
@ashkarpetin Added, thank you.
@noon-dawg Applied one of your suggestions, thank you!

File: openapi/components/schemas/Storefront/Account.yaml [hunk 2]
```diff
@@ -43,3 +43,13 @@ properties:
     items:
       anyOf:
         - $ref: ../Links/SelfLink.yaml
+  _embedded:
+    type: array
+    description: >-
```
Comment (ashkarpetin) [inline openapi/components/schemas/Storefront/Account.yaml:48]:
```suggestion
    description: |-
```

**Review by ashkarpetin — APPROVED** [review]

**Comment by noon-dawg** [comment]
Looks good to me!

**Review by VladimirBerdnik — APPROVED** [review]

**Pull request merged**